dashboard/db/migrate/20160802170542_add_readonly_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddReadonlyUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
+  end
+end
```

Comment:
This is a big migration that we probably want to run separately from this rest of this PR once ready. @breville @ashercodeorg had some thoughts/experience on how to best manage this.